packages/puppeteer-core/src/bidi/Page.ts

```diff
@@ -674,14 +692,12 @@ export class BidiPage extends Page {
       result: {data},
     } = await this.#connection.send('browsingContext.captureScreenshot', {
       context: this.mainFrame()._id,
+      origin: captureBeyondViewport ? 'document' : 'viewport',
       format: {
         type: `image/${type}`,
-        quality: quality ? quality / 100 : undefined,
-      },
-      clip: clip && {
-        type: 'box',
-        ...clip,
+        ...(quality !== undefined ? {quality: quality / 100} : {}),
```

Comment:
The `quality` and `box` property are hard to read, please consider either the original or an alternative.

Reply:
This is the only way to ensure the keys are not specified if they do not exist.